packages/mesa.rb

```diff
@@ -26,9 +26,13 @@ class Mesa < Package
   depends_on 'libxtrans'
   depends_on 'libxkbfile'
   depends_on 'wayland'
+  depends_on 'python'
+  depends_on 'bison'
 
+  # tested on armv7l
   def self.build
-    system "./configure --enable-gles2 --with-egl-platforms=x11,wayland,drm --enable-gbm --enable-shared-glapi --with-gallium-drivers=swrast --disable-dri3"
+    system "pip install Mako"
+    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} --enable-shared-glapi --with-gallium-drivers=nouveau --with-dri-drivers=nouveau --enable-egl --with-platforms=x11,drm,wayland --enable-gbm --enable-xa --enable-dri"
```

Comment:
It would be good for readability if you put each option on a separate line.  Something like:
```
system "./configure \
  --prefix=#{CREW_PREFIX} \
  --libdir=#{CREW_LIB_PREFIX} \
  --enable-shared-glapi \
  --with-gallium-drivers=nouveau \
  --with-dri-drivers=nouveau \
  --enable-egl \
  --with-platforms=x11,drm,wayland \
  --enable-gbm \
  --enable-xa \
  --enable-dri"
```
This also makes it easier to add or remove options simply by adding/removing the option line.